tools/build/src/commands/docs.ts

```diff
@@ -175,6 +179,7 @@ const build = async (lang: string) => {
 
   const docsJson = {
     name: `${lang}`,
+    badge: `${badge}`,
```

Comment:
No need to nest this in a string, you can just do:
```
badge: badge,
```

And then you can use property shorthand, so just:
```
badge,
```